WikiTemplate/Crypto/wikitemplate-cryptoAndroid.md

```diff
@@ -0,0 +1,59 @@
+
+## Onboarding
+
+- [ ] Verify able to create a wallet by following onboarding
+- [ ] Verify not able to screenshots during onboarding screens
+- [ ] Verify able to backup wallet seed phrase during onboarding
+- [ ] Verify able to skip backup wallet seed screen to complete onboarding
+- [ ] Verify onboarding is shown when trying to interact with Dapp without creating wallet
+
+## Panel
+
+- [ ] Verify panel is only shown in URL when a Dapp is trying to interact
+- [ ] Verify able to change default network from panel
+- [ ] Verify able to change accounts from panel
+- [ ] Verify notification dot is shown when panel when an action is pending for Wallet
```

Comment:
`Verify notification dot is shown when panel when an action is pending for Wallet` ->
`Verify notification dot is shown on a wallet icon when an action is pending for Wallet`
?

Reply:
Updated